coral/src/app/features/browse-topics/BrowseTopics.tsx

```diff
@@ -38,12 +38,12 @@ function BrowseTopics() {
   return (
     <>
       <Flexbox direction={"column"} gap={"l1"}>
-        <Flexbox justifyContent={"end"} colGap={"l1"} maxWidth={"6xl"}>
+        <Flexbox justifyContent={"end"} colGap={"l1"}>
```

Comment:
I know I was the one suggesting aligning to the right, but after seeing it I feel it might be better on the left, sorry for the wishywashiness 😬

Reply:
:D 

Not a problem. 

So, I reverted the commit, it's that what you're saying we should do?